### PR TITLE
Add Player::SetRoom

### PR DESCRIPTION
**Pull request by RedRangerr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <img src="https://img.shields.io/discord/656673194693885975?logo=discord" alt="Chat on Discord">
 </a>
 
-
+
```
Comment (PintTheDragon) [inline README.md:10]: Why?

**Review by PintTheDragon — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RedRangerr) [reply README.md:10]: i had to trigger the build

**Comment by RedRangerr** [comment]
its fine now?

**Comment by RedRangerr** [comment]
there pint

**Review by joker-119 — CHANGES_REQUESTED** [review]
Why is this it's own method, instead of adding a setter to Player::CurrentRoom ?
You will also need to compensate for the fact that some rooms centers don't put you inside the room itself, or put you in a place you will die or be out of bounds.  For this you should use the base-game ForceSafePos method, or some up with your own way to compensate for specific rooms that fall under the above issues, and offset the position the player is sent to based on the rooms rotation etc.

As it is, I can think of 5 rooms off the top of my head, that this will break for.

**Comment by NaoUnderscore** [comment]
@RedRangerr Close this PR, it's pointless, unless you know what and how you're doing. It's going to break shit with a certain rooms.

**Pull request closed without merging**